cmd/platform/user.go

```diff
@@ -202,7 +201,7 @@ func userDeactivateCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) < 1 {
-		return errors.New("Enter user(s) to deactivate.")
+		return errors.New("Expected at least one argument. See help text for details.")
```

Comment:
The help text that I'm referring to is what is printed if the user specifies the --h flag. There might be online documentation, but if there is, I'm not familiar with it. Users that are familiar with command line tools typically expect a man page or a --h flag to pull up documentation.